### ISSUE TITLE
Doc about k8s CRD session sticky

### ISSUE THREAD

**Issue by zyf0330** [containous/traefik#6133]
### Do you want to request a *feature* or report a *bug*?

Feature

### What did you expect to see?

PR #5711 added support for k8s CRD session sticky. But I cannot find any doc to set it.

**Comment by zyf0330** [comment]
After some test, I find solution code. This is example used in `IngressRoute`.
```
  - match: PathPrefix(`/ping`)
    kind: Rule
    services:
    - name: name
      port: 8888
      sticky:
        cookie:
          httpOnly: true
```

There are also some examples of `kind: TraefikService`at docs/content/reference/dynamic-configuration/kubernetes-crd.yml for advanced usage.
Still find nothing in official website doc.

**Comment by fabio-510** [comment]
Somewhat related to this I cannot find documentation on how to properly configure sticky sessions with standard Kubernetes definitions, like Ingress and Service, rather than IngressRoute or TraefikService.
The main motivation is that I have other services that work with the k8s provided objects rather than traefik (like linkerd and cert-manager).

Documentation is very confusing and it just makes me circle around while trying to achieve what I am trying to achieve.

From my current understanding I can achieve sticky sessions by either using Traefik's custom CRDs or File Providers. The custom CRDs are not an option and File Provider documentation is not very clear on how to be setup using Kubernetes objects (like configmaps or volume mounts), so I have been running in circles.

Ideally I would like to have sticky sessions with standard Kubernetes objects, but in an absense of that, file providers will do. But that's not very clear on how to achieve using the guide here: https://docs.traefik.io/v2.1/user-guides/crd-acme/

I tried using 1.7s helm chart but it didn't quite work for several reasons, that's why I set up the 2.1, using the guide, which actually worked out of the box for everything I need except sticky sessions (which are necessary because of websocket connections).

Maybe I am missing a piece of documentation, but I believe it's actually lacking. Partly on sticky sessions and partly on configuration.

Apologies if am in the wrong place here.
